bliss/encoder/encoder.py

```diff
@@ -10,13 +10,14 @@
 from torch.optim.lr_scheduler import MultiStepLR
 from torchmetrics import MetricCollection
 
-from bliss.catalog import TileCatalog
 from bliss.encoder.convnet import CatalogNet, ContextNet, FeaturesNet
 from bliss.encoder.data_augmentation import augment_batch
 from bliss.encoder.image_normalizer import ImageNormalizer
 from bliss.encoder.metrics import CatalogMatcher
 from bliss.encoder.plotting import plot_detections
 from bliss.encoder.variational_dist import VariationalDistSpec
+from case_studies.redshift_estimation.catalog import RedshiftTileCatalog
```

Comment:
This seems backwards. We want the case studies to import the bliss package, not the other way around.